plasmapy/physics/tests/test_transport.py

```diff
@@ -407,7 +407,7 @@ def test_electron_thermal_conductivity_by_model(self, model, expected):
                            atol=1e-6*u.W / (u.K * u.m))
 
     @pytest.mark.parametrize("model, expected", [
-        ("ji-held", 115593.62970243 * u.W / (u.K * u.m)),
+        ("ji-held", 131600.515524444 * u.W / (u.K * u.m)),
```

Comment:
Wait, and these tests are still passing? What the hell are we checking there... is `atol=1e-6*u.W / (u.K * u.m))` a good fit here?

Reply:
So on this one, these tests started failing when I changed the default behavior of the ji-held model to use mu=0. Then, I changed the expected value in the test to match what it was outputting, so the test would pass again.

Yeah, for these tests with the high precision, I'm like, certainly we don't mean to say that we know a "correct" transport result to such high accuracy, when the models that we are using are only written for 1% accuracy or so at best. But at the same time, these tests seem like they'd be handy for code refactoring, for instance if we're shuffling a bunch of things around intending not to alter the function at all, and we do by mistake, then tests like this will assuredly catch even the smallest change. Maybe we should mark them as non-critical somehow?

Reply:
Oh crud, I'm sorry, I completely missed the fact that changing the default mu would change this value. No idea how that happened. This test is completely fine, then, I think. Maybe we could add a comment that we're comparing these values to results this very code gave us previously, as opposed to values from Braginskii?